packages/integrations/cloudflare/src/index.ts

```diff
@@ -97,7 +97,7 @@ export default function createIntegration(args?: Options): AstroIntegration {
 
 				await esbuild.build({
 					target: 'es2020',
-					platform: 'browser',
+					platform: 'node',
```

Comment:
This isn't right. The Cloudflare environment is not node, but workers, which the closest platform is `browser`. Otherwise you'd get `fs`, `path` etc modules bundled, which woudln't work in Cloudflare.

It doesn't look like lit supports worker environments too: https://github.com/lit/lit/issues/1983

Reply:
>  Cloudflare environment is not node

While this is true, two different lit packages do not build correctly for Cloudflare with platform set to `browser`. The correct (node) versions of the broken lit packages actually compile correctly with this setting (and don't include node libs).

I've tried `platform: 'neutral', conditions: ['node']`; that attempts to build the node versions of `@lit/reactive-element` and `lit-html` (the problem packages) in a more V8-like context, but the build fails.

I'll see if there's any way to get Lit to build correctly with `platform: 'neutral'`, because, as of now, Lit is completely broken on Cloudflare.

Reply:
@bluwy , I've updated this to something that might be more amenable to Cloudflare's Worker environment. Do you know if this would break any other integrations? The tests for this package passed, but I'm not sure how deep the tests go for integrations.